tests/src/Context/SchedulerContext.php

```diff
@@ -39,4 +58,33 @@ public function iChangeUnpublishingDateOfTypeNodeWithTitleToDate($type, $title,
     }
   }
 
+  /**
+   * Adds the current day with format Y-m-d.
+   *
+   * @Then I fill in :arg1 with current day
+   */
+  public function iFillInWithCurrentDay($arg1) {
```

Comment:
This two steps seem to be unused so I guess we should delete them.

Reply:
We use them now in the scenario to check the publication of revision